configs/neox_arguments.md

```diff
@@ -7,7 +7,7 @@ LR Scheduler Arguments
 
 
 
-- **lr_decay_style**: typing.Literal['constant', 'linear', 'cosine', 'exponential']
+- **lr_decay_style**: Literal
```

Comment:
Just confirming these changes to `neox_arguments.md` auto-update are desired?

Reply:
They're not desired, and I have no idea why they're being triggered tbh. Looking into it.